docs/_docs/contributing/code.md

```diff
@@ -12,3 +12,55 @@ Generally, for code contribution you should:
 4. Open a pull request against the master branch.
 
 See the repository `CONTRIBUTING.md` for these same details.
+
+## Contributing a Custom Function
+
+Deid ships (as of version 0.2.3) with deid-provided functions that can be used in
+header parsing. To contribute a recipe you should do the following:
```

Comment:
Trivial comment - but this should probably read: "To contribute a **custom function**..."

@vsoch - Sorry for my delay looking at this and responding. I've continued to be pulled in other areas so haven't had much time to spend on deid...

Reply:
Totally no worries! I'll be around when you are ready :)